app/javascript/mastodon/reducers/accounts_counters.js

```diff
@@ -126,6 +126,7 @@ export default function accountsCounters(state = initialState, action) {
   case STATUS_FETCH_SUCCESS:
     return normalizeAccountFromStatus(state, action.status);
   case ACCOUNT_FOLLOW_SUCCESS:
+    if (action.alreadyFollowing) { return state; }
```

Comment:
If blocks within a switch are kind of confusing. A ternary probably *is* better, and if it's less readable I think you can multiline it.